src/commands.rs

```diff
@@ -1314,6 +1315,25 @@ struct ObslogArgs {
     diff_format: DiffFormatArgs,
 }
 
+/// Compare the changes of two commits
+///
+/// This excludes changes from other commits by temporarily rebasing `--to` onto
+/// `--from`'s parents. If you wish to compare the same change across versions,
```

Comment:
I think it's better to rebase in the other direction. That mostly matters if there are conflicts. Let's say you have rebased A from B onto C as A', which resulted in conflicts. You then resolved the conflicts, creating A''. If you now do `jj interdiff --from A --to A''`, you'd probably expect it to show you a diff where the "left" side has a conflict and the "right" side has the contents from A''. If you rebase the `--to` side instead, you'll see a diff with the contents from A on the left and a conflict on the right.

Reply:
Thanks, sounds good. I should have mentioned in the PR description that the from/to concept has never made sense to me, so this part wasn't very thought out..